src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -103,35 +103,31 @@ export const postSubmit = async (req: Request, res: Response, next: NextFunction
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const appData: ApplicationData = getApplicationData(req.session);
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS) && checkEntityRequiresTrusts(appData)) {
```

Comment:
I don't think you should move this section to the top as it affects the redirect-ing.
Previously, if hasTrustsToReview would redirect back to the start of the trusts pages before the checkEntityRequiresTrusts redirect.

Now it will perform the checkEntityRequiresTrusts redirect first, which will alter the behaviour of the page

Reply:
i would just reset this method and just remove the if check for the feature flag we are removing

Reply:
Reset this.